transforms/no-implicit-ref-callback-return.js

```diff
@@ -0,0 +1,38 @@
+const parseSync = require("./utils/parseSync");
+
+/**
+ * @type {import('jscodeshift').Transform}
+ */
+const noImplicitRefCallbackReturnTransform = (file, api) => {
+	const j = api.jscodeshift;
+	const ast = parseSync(file);
+
+	let changedSome = false;
+
+	ast
+		.find(j.JSXAttribute, (jsxAttribute) => {
+			return jsxAttribute.name.name === "ref";
+		})
+		.forEach((jsxAttributePath) => {
+			const jsxAttribute = jsxAttributePath.node;
+			if (
+				jsxAttribute.value?.type === "JSXExpressionContainer" &&
+				jsxAttribute.value.expression.type === "ArrowFunctionExpression" &&
+				jsxAttribute.value.expression.body.type !== "BlockStatement"
+			) {
```

Comment:
Ideally this would be part of the `filter` but this way TypeScript automatically knows what type we're dealing with.